docs/griptape-cloud/getting-started/quickstart-cli.md

```diff
@@ -36,13 +36,13 @@ Next, we can use the Griptape CLI to scaffold a sample project that is compatibl
 You should see something like the following. In this example, I'm creating an app called _demo_app_ in my Home directory. I'm then verifying its contents via `ls`.
```

Comment:
> In this example, I'm creating an app called _demo_app_ in my Home directory

Might want to mention the `workplace` directory.

Reply:
Good catch, updated